modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/VerticalNavTag.java

```diff
@@ -53,6 +57,10 @@ public List<VerticalNavItem> getVerticalNavItems() {
 		return _verticalNavItems;
 	}
 
+	public void setActive(String active) {
```

Comment:
maybe we shall name this `activeKey` to ditinguish it from the `active` flag in vertical navigation item class, as well as from the `active` internal variables we use when rendering. This will leak to the tag api.

Reply:
I think it's fine to name the attribute `active`, as it matches [Clay API](https://github.com/liferay/clay/blob/master/packages/clay-core/src/vertical-nav/VerticalNav.tsx#L38-L41). What we should be doing is passing it on to Clay, like in the frontend part.

What looks wrong in this PR is the internal logic with `active` as a boolean. There is no need for it.

Reply:
let me know!

Reply:
I've added the prefix as Marko suggested https://github.com/liferay-frontend/liferay-portal/pull/3475/commits/3f36a88218ca5cb24ef1003b05fa603d8234f700